Symfony/CS/Fixer/Symfony/PhpdocShortDescriptionFixer.php

```diff
@@ -13,6 +13,7 @@
 
 use Symfony\CS\AbstractFixer;
 use Symfony\CS\DocBlock\DocBlock;
+use Symfony\CS\DocBlock\Line;
```

Comment:
Importing this has overhead for the runtime performance - I think we should just the the FQCN in the phpdoc to avoid this import.